jscomp/super_errors/super_typecore.ml

```diff
@@ -63,11 +137,7 @@ let report_error env ppf = function
           If so, please use `ReasonReact.createDomElement`:@ https://reasonml.github.io/reason-react/docs/en/children.html@]@,@,\
           @[@{<info>Here's the original error message@}@]@,\
         @]";
-      super_report_unification_error ppf env trace
-        (function ppf ->
-           fprintf ppf "This is:")
-        (function ppf ->
-           fprintf ppf "But somewhere wanted:")
+      helpful_unification_error ppf env trace
```

Comment:
Not too sure; here I'm trying to stay with the structure of the original `typecore.ml`, so I'd rather inline this. No strong opinion though